manage.py

```diff
@@ -904,33 +895,189 @@ def _add_lambda_subparser(subparsers):
     # Set the name of this parser to 'lambda'
     lambda_parser.set_defaults(command='lambda')
 
-    # Add subcommand options for the lambda sub-parser
-    lambda_parser.add_argument(
-        'subcommand',
-        choices=['deploy', 'rollback', 'test'],
+    lambda_subparsers = lambda_parser.add_subparsers()
+
+    _add_lambda_deploy_subparser(lambda_subparsers)
+    _add_lambda_rollback_subparser(lambda_subparsers)
+    _add_lambda_test_subparser(lambda_subparsers)
+
+
+def _add_lambda_deploy_subparser(lambda_subparsers):
+    """Add the lambda deploy subparser: manage.py lambda deploy"""
+    lambda_deploy_usage = 'manage.py lambda deploy'
+
+    lambda_deploy_desc = ("""
+StreamAlertCLI v{}
+Deploy Lambda functions
+
+Command:
+
+    manage.py lambda deploy            Deploy Lambda functions
+
+Required Arguments:
+
+    -p/--processor                     The name of the Lambda function to deploy.
```

Comment:
how does this change deployment logic? was this tested?

Reply:
There is no change to how these commands work and everything still functions as it did before, testing the command works fine